### PR TITLE
:paperclip: Add more events instrumentation

### PR DESCRIPTION
**Pull request by niwinz**

### CODE REVIEW

File: frontend/src/app/main/ui/auth/register.cljs [hunk 1]
```diff
@@ -160,13 +161,6 @@
 (defn- handle-register-error
   [form error]
   (case (:code error)
-    :registration-disabled
-    (st/emit! (dm/error (tr "errors.registration-disabled")))
-
-    :email-has-permanent-bounces
-    (let [email (get @form [:data :email])]
-      (st/emit! (dm/error (tr "errors.email-has-permanent-bounces" email))))
-
```
Comment (hirunatan) [inline frontend/src/app/main/ui/auth/register.cljs:169]: Why are those removed?

Comment (niwinz) [reply frontend/src/app/main/ui/auth/register.cljs:169]: because they are not handled here, so this is technically dead code.
